catalog/dags/providers/provider_api_scripts/rawpixel.py

```diff
@@ -132,6 +135,24 @@ def get_batch_data(self, response_json):
             return results
         return None
 
+    @staticmethod
+    def _get_full_size_preset(self, data: dict) -> str | None:
+        """Get the full size preset based on the image properties."""
+        if data.get("isPng") and data.get("isDarkPng"):
+            return RawpixelDataIngester.png_dark_full_size_option
+        if data.get("isPng"):
+            return RawpixelDataIngester.png_full_size_option
+        return RawpixelDataIngester.full_size_option
```

Comment:
```suggestion
        if data.get("isPng"): 
            if data.get("isDarkPng"):
                return RawpixelDataIngester.png_dark_full_size_option
            return RawpixelDataIngester.png_full_size_option
        return RawpixelDataIngester.full_size_option
```
This is a bit of a simplification that avoids having to repeat the condition!